packages/metal-dom/src/events.js

```diff
@@ -2,6 +2,45 @@
 
 import { registerCustomEvent } from './dom';
 import features from './features';
+import * as KEYMAP from './keyConstants';
+
+const REGEX_EVENT = /^(key+(down|up|press)):((,?[a-z])+)/;
+
+/**
+ * Returns the relative keyCode from each key name in an array.
+ * @param {Array<string>} keyNames An array with the key names.
```

Comment:
This is missing the `@return` doc. Also, the description says that this will return the result `in an array`, but it's actually `as an object`.